apps/cloud_composer/src/cloud_view.cpp

```diff
@@ -42,7 +42,8 @@ pcl::cloud_composer::CloudView::CloudView (ProjectModel* model, QWidget* parent)
 }
 
 pcl::cloud_composer::CloudView::CloudView (const CloudView& to_copy)
-  : vis_ (to_copy.vis_)
+  : QWidget()
```

Comment:
Maybe including `to_copy` in [base class copy ctor](https://stackoverflow.com/a/17311455) might be useful

Reply:
The base class (QWidget) copy constructor is private. This is the only way AFAIK.

Reply:
You are indeed correct. [Docs](https://doc.qt.io/qt-5/qobject.html#no-copy-constructor-or-assignment-operator) tell us to use a pointer instead of subclassing 🤷‍♂ 

That's for a separate issue though